static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -230,6 +231,29 @@ export function StacktraceLink({frame, event, line, disableSetup}: StacktraceLin
     refetch();
   };
 
+  const handleCopyPath = async (e: React.MouseEvent) => {
+    e.stopPropagation();
+    if (!frame.filename || !frame.lineNo) {
+      return;
+    }
+
+    const filePath = frame.filename + ':' + frame.lineNo;
+
+    trackAnalytics('integrations.stacktrace_file_path_copied', {
+      view: 'stacktrace_issue_details',
+      organization,
+      group_id: event.groupID ? parseInt(event.groupID, 10) : -1,
+      ...getAnalyticsDataForEvent(event),
+    });
+
+    try {
+      await navigator.clipboard.writeText(filePath);
+      addSuccessMessage(t('File path copied to clipboard'));
+    } catch (err) {
+      addErrorMessage(t('Failed to copy file path'));
+    }
+  };
```

Comment:
I'd recommend using the `useCopyToClipboard` from [`static/app/utils/useCopyToClipboard.tsx`](https://github.com/getsentry/sentry/blob/master/static/app/utils/useCopyToClipboard.tsx) hook here!